app/src/main/res/values-night/colors.xml

```diff
@@ -126,8 +126,6 @@
     <color name="add_on_private_browsing_exterior_circle_background_normal_theme">@color/accent_normal_theme</color>
     <color name="add_on_private_browsing_interior_icon_background_normal_theme">@color/photonWhite</color>
     <color name="prompt_login_edit_text_cursor_color_normal_theme">@color/photonViolet50</color>
-    <color name="readermode_start_gradient_normal_theme">@color/photonViolet30</color>
-    <color name="readermode_end_gradient_normal_theme">@color/photonBlue30</color>
```

Comment:
This is intentional that we are going with the Violet20 and Blue20 for the dark theme to align with the desktop gradient.